### DISCUSSION TITLE
"Cannot read properties of undefined (reading 'split')" after first message

### DISCUSSION THREAD

**Discussion by OccultSlolem** [community/community#102408] [Copilot Conversations]
### Select Topic Area

Bug

### Body

I'm incapable of carrying on conversations with Copilot due to this bug. Any message after the first will always have this issue. This started happening on the pre-release version of the extension maybe a week or two ago, I tried going to the release version but it's still broken.

![Example](https://descriptive-eagle-550.convex.cloud/api/storage/28bb324b-682e-4250-9385-48f2e9d1bb52)

**Comment by naveen-98** [comment]
Hey @OccultSlolem !  😀

I understand that this bug is preventing you from having productive conversations with GitHub Copilot, and that it began on the pre-release version and persists even after updating to the release version. 

Here are some things you can try:

1. Disable and re-enable GitHub Copilot:

In your text editor, go to Settings > Extensions.
Find the GitHub Copilot extension and disable it.
Restart your text editor.
Re-enable the GitHub Copilot extension.
2. Clear GitHub Copilot's cache:

Open a terminal window.
Run the following command: gh copilot auth logout
Run the following command: gh copilot auth login
3. Report the bug:

If the above steps don't resolve the issue, please report the bug to the GitHub Copilot team. You can do this by:
Clicking on the feedback button within the GitHub Copilot extension.
Creating an issue on the GitHub Copilot repository: https://github.com/microsoft/vscode-copilot-release/issues
4. Use an alternative to GitHub Copilot:

While not ideal, you can consider using an alternative code completion tool such as TabNine or Kite while the GitHub Copilot bug is being fixed.
Additional tips:

Make sure you are using the latest version of GitHub Copilot.
Check if there are any known issues or workarounds mentioned in the GitHub Copilot documentation: https://docs.github.com/copilot
You can also search online for other users who have encountered this bug and see if they have found any solutions.
I hope one of these suggestions helps you resolve the issue and get back to having productive conversations with GitHub Copilot.

**Reply by OccultSlolem** [reply]
`gh copilot ...` does not appear to be a command 🙃

**Reply by naveen-98** [reply]
I understand you're encountering the error "gh copilot ... does not appear to be a command." Let's check a few things to ensure the command is available:

1. Install the GitHub CLI:

If you haven't already, download and install the GitHub CLI from the official website: https://cli.github.com/
Run the installer and follow the prompts.
2. Verify Installation:

Open a terminal window and try running gh --version. This should display the installed version of the GitHub CLI.
If not installed, re-run the installer or seek assistance from the GitHub CLI documentation.
3. Check PATH:

Ensure the directory containing the GitHub CLI executables is included in your system's PATH environment variable.
Instructions vary by operating system. Consult documentation for your OS if needed.
You can usually check the PATH by running echo $PATH in your terminal.
4. Restart Terminal:

After installation or PATH adjustments, close and reopen your terminal window.
5. Reattempt Commands:

Once the GitHub CLI is correctly installed and accessible, try running gh copilot auth logout and gh copilot auth login again.
If the issue persists, please provide more details about your environment:

Operating system (Windows, macOS, Linux):
Terminal app being used (Command Prompt, PowerShell, Terminal, etc.):
Any specific error messages encountered:
I'm here to assist further based on the additional information you provide.

Remember: It's essential to have the GitHub CLI installed correctly for the "gh copilot" commands to function.

**Reply by OccultSlolem** [reply]
Bro, I appreciate what you're trying to do here, but I already asked ChatGPT and Copilot if they could resolve the situation.

**Reply by naveen-98** [reply]
Finally, I also tried this with CHATGPT. But its answers are not successful.

**Comment by elushnikova** [comment]
@OccultSlolem, hi there, the same happened right now after my VS Code updated.
In the Extensions I saw "Reload" button on Copilot. After reloading it I was able to proceed with conversations past the first message.

I know nothing about the versions involved, but I hope that helps.
Please, let me know if additional info could be helpful.
